### PR TITLE
Remove gc and update Watcher

### PR DESCRIPTION
**Pull request by irisradoi**
Related to https://github.com/PISAresearch/pisa/issues/218

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
         "express": "^4.16.4",
         "express-http-context": "^1.2.0",
         "express-rate-limit": "^3.5.0",
-        "ganache-core": "^2.5.6",
```
Comment (yahgwai) [inline package.json]: You should revert this change. We try to keep packages the same, then upgrade them explicitly as part of an issue.

You can use `npm ci` to install the exact versions that are in the package-lock.json

File: src/watcher/watcher.ts [hunk 2]
```diff
@@ -121,6 +121,15 @@ export class Watcher extends Component<WatcherAnchorState, IBlockStub & Logs> {
         appointmentState.state === WatcherAppointmentState.OBSERVED &&
         state.blockNumber - appointmentState.blockObserved + 1 >= this.confirmationsBeforeRemoval;
 
+    private shouldRemoveExpiredAppointment = (
```
Comment (yahgwai) [inline src/watcher/watcher.ts]: Can you change the name of `shouldRemoveAppointment` to `shouldRemoveObservedAppointment` to avoid ambiguity.

Comment (irisradoi) [reply src/watcher/watcher.ts]: Done

File: package.json (hunk 1 above)
Comment (irisradoi) [reply package.json]: Done

**Pull request merged**